health_check/contrib/rabbitmq/backends.py

```diff
@@ -0,0 +1,86 @@
+import logging
+
+import pika
+from django.conf import settings
+from pika.exceptions import ConnectionClosed, ProbableAuthenticationError
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceUnavailable
+
+logger = logging.getLogger(__name__)
+
+
+class RabbitMQHealthCheck(BaseHealthCheckBackend):
+    """
+    Healthchecker for RabbitMQ.
+
+    """
+
+    def __get_broker_url(self):
+        """
+        Gets the broker url based on django.confg.settings dictionary.
+
+        """
+        broker_url = getattr(settings, "BROKER_URL", None)
+
+        if broker_url is None:
```

Comment:
I have the feeling this drives complexity. I'd rather have only one approach on how to configure this correctly.